lib/protocol/bep.proto

```diff
@@ -100,6 +100,7 @@ message FileInfo {
     uint32       permissions    = 4;
     int64        modified_s     = 5;
     int32        modified_ns    = 11;
+    string       modified_by    = 13;
```

Comment:
Well I don't know if I can do that.  I had short ID before but you guys wanted friendly name and that means I need the full ID.   I tried using []byte with protobuf but it was giving me wierd errors so I had to compromise with it and just go with string (of full ID).